cmd/mayactl/app/command/volume_info.go

```diff
@@ -250,14 +268,60 @@ Replica Count :   {{.ReplicaCount}}
 
 		// parsing the information to replicastatus template
 		tmpl = template.New("ReplicaInfo")
-		tmpl = template.Must(tmpl.Parse(replicaTemplate))
+		tmpl = template.Must(tmpl.Parse(jivaReplicaTemplate))
 
 		w := tabwriter.NewWriter(os.Stdout, v1.MinWidth, v1.MaxWidth, v1.Padding, ' ', 0)
 		err = tmpl.Execute(w, replicaInfo)
 		if err != nil {
 			fmt.Println("Unable to display volume info, found error : ", err)
 		}
 		w.Flush()
+	} else {
```

Comment:
Lets check specifically for both jiva and cstor types instead of making cstor as default for else.

Reply:
ohk